### ISSUE TITLE
feedback/voting  for the items  to make good content visible 

### ISSUE THREAD

**Issue by shel3over** [01walid/dzlibs#5]
i can see that after few month the dzlibs will have lots of contribution
be able to evaluate the items helps good content to be visible

**Comment by 01walid** [comment]
Right,
I was thinking about an appreciate button (like Behance), what do you think ?

**Comment by shel3over** [comment]
@01walid  rating ( 5 star for example ) + comment as feedback s 
give more information why u  think this item is recommended or not

**Comment by mohamed-io** [comment]
What about "I use this asset" button? i think it'll be a good usage indicator.

**Comment by assem-ch** [comment]
what about using the "Watch/Star/fork" of github

**Comment by 01walid** [comment]
For Github-based items, embedding github buttons (fork/star/watch) is already planned, I'll work on it soon.

For the voting part, I tend to like Mohamed's idea for now, it's really suitable to the nature of the website

beside that, do you think embedding Disqus comment system is worth it as well ?

**Comment by mohamed-io** [comment]
Comment system is a must-have feature, I guess.

**Comment by shel3over** [comment]
+1 for the Disqus :cat2:

**Comment by breda** [comment]
Yes! I believe Disqus is the best solution… That is, if you guys want to use third-party tools.

**Comment by zirconias** [comment]
+1 for disqus saves lot of work

**Comment by 01walid** [comment]
Disqus implemented now.
next, Github buttons ...
